pyaedt/modules/Material.py

```diff
@@ -446,8 +428,8 @@ def add_thermal_modifier_closed_form(self, tref=22, c1=0.0001, c2=1e-6, tl=-273.
 
         Returns
         -------
-        bool
-            ``True`` when successful, ``False`` when failed.
+        type
```

Comment:
```suggestion
        bool
            ``True`` when successful, ``False`` when failed.
```
Same reason as above.